pkgs/tools/admin/otc-auth/default.nix

```
@@ -0,0 +1,31 @@
+{ pkgs, lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "otc-auth";
+  version = "2.0.3";
+
+  src = pkgs.fetchFromGitHub {
+    rev = "v${version}";
+    owner = "iits-consulting";
+    repo = "otc-auth";
+    sha256 = "sha256-dKBsceTv4SW3YWuVtC7U4u0IY9VZOrJ+QM5Jq+DW4Hw=";
```

```suggestion
    hash = "sha256-dKBsceTv4SW3YWuVtC7U4u0IY9VZOrJ+QM5Jq+DW4Hw=";
```

You should use hash instead of sha256 because the value is a SRI hash here.